features/steps/steps.py

```diff
@@ -36,11 +36,38 @@ def when_i_run_command(context, command, user_spec):
     context.process = process
 
 
+@when('I attach "{token}" {user_spec}')
+def when_i_attach_token(context, token, user_spec):
+    token = context.config.contract_token
+    if not token:
+        context.scenario.skip(
+            "Skipping test because you didn't provide a token"
+        )
+    when_i_run_command(context, "ua attach %s" % token, user_spec)
+
+
 @then("I will see the following on stdout")
 def then_i_will_see_on_stdout(context):
     assert_that(context.process.stdout.strip(), equal_to(context.text))
 
 
+@then("stdout will include")
+def then_and_stdout_will_include(context):
```

Comment:
typo fix function name, can drop "_and"

Reply:
I can drop this hook and use just the one with `matches_regexp` instead of it + `contains_string`